course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -251,6 +251,12 @@ def format_course_run_data(self, body, course=None):
         if course:
             defaults['course'] = course
 
+        course_detail = self.api_client.courses(body['id']).get()
```

Comment:
Hi @huynguyen-groove ,

Obviously, we can get overview by send a request to LMS course API. However, that's a bad idea though we can add additional property overview into the CourseSerializer, why not?

edx-platform/lms/djangoapps/course_api/serializers.py line 83